src/Pages/Sign/Login.js

```diff
@@ -67,14 +71,14 @@ class Sign extends Component {
               </div>
             </div>
             <div className="signInMenu">
-              <div className="descriptionBox">
+              <div className="descriptionBox">np
                 <p className="description">
                   아직 회원이 아니신가요?
                   <br></br>
                   가입 후 쿠폰 및 포인트 적립 등의 다양한 헤택을 받아보세요.
                 </p>
               </div>
-              <button className="signUpButton">회원가입</button>
+              <Link to="/signUp"><button className="signUpButton">회원가입</button></Link>
```

Comment:
```suggestion
              <Link to="/signUp">
                <button className="signUpButton">회원가입</button>
              </Link>
```
부모 자식 요소가 잘 드러나도록 줄바꿈 해주세요